### PR TITLE
KT-25270 "return@foo" outside of lambda should have quickfix to remove "@foo" label

### PR DESCRIPTION
**Pull request by t-kameyama**
https://youtrack.jetbrains.com/issue/KT-25270

### CODE REVIEW

File: idea/testData/inspectionsLocal/redundantReturnLabel/inLambda.kt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// PROBLEM: none
+fun foo(f: (String?) -> Int) {}
+
+fun test() {
+    foo {
+        if (it != null) return@foo<caret> 1
```
Comment (udalov) [inline idea/testData/inspectionsLocal/redundantReturnLabel/inLambda.kt]: Please make sure that the inspection is not suggested when using the function expression syntax:
```
foo(fun(it: String?): Int {
    if (it != null) return@foo 1
    return 0
})
```

Comment (t-kameyama) [reply idea/testData/inspectionsLocal/redundantReturnLabel/inLambda.kt]: Why we should not suggest?

Comment (udalov) [reply idea/testData/inspectionsLocal/redundantReturnLabel/inLambda.kt]: Because this is valid code, unlike in the issue fixed in this pull request

**Comment by mglukhikh** [comment]
Generally looks good, I will test the inspection a bit and then fix / merge it.

**Comment by mglukhikh** [comment]
Finally merged, thank you

**Pull request closed without merging**